internal/dataexchange/ffdx/ffdx.go

```diff
@@ -347,6 +349,8 @@ func (h *FFDX) eventLoop() {
 			case messageFailed:
 				err = h.callbacks.TransferResult(msg.RequestID, fftypes.OpStatusFailed, fftypes.TransportStatusUpdate{Error: msg.Error})
 			case messageDelivered:
+				err = h.callbacks.TransferResult(msg.RequestID, fftypes.OpStatusSucceeded, fftypes.TransportStatusUpdate{})
```

Comment:
If this is received when `manifest` mode is `true` - should this not still be `OpStatusPending`?

Reply:
Good point, the logic has been updated to keep the operation status as "Pending" at this stage.